app/layout/nav_menu.phtml

```diff
@@ -175,6 +175,10 @@
 		</a>
 	</div>
 
+	<div class="stick" id="nav_menu_hooks">
```

Comment:
Ups, I did not notice at first, but this made a layout change:

Before:

![image](https://user-images.githubusercontent.com/1008324/104251332-c5b4de00-546f-11eb-9a9f-ffb14786c152.png)

After:

![image](https://user-images.githubusercontent.com/1008324/104251359-d5342700-546f-11eb-9e98-4c067ede81bb.png)

Could you please try to fix @cn-tools ?

Reply:
ups ... please have a look at #3362